synapse/http/server.py

```diff
@@ -255,12 +257,12 @@ def __init__(self, hs, canonical_json=True):
         self.path_regexs = {}
         self.hs = hs
 
-    def register_paths(self, method, path_patterns, callback):
+    def register_paths(self, method, path_patterns, callback, servlet_classname):
```

Comment:
could probably do with taking the opportunity to add a docstring here. What does this method do, what do its params do, and what are their expected types?